docs/v1.0/parser_nginx.txt

```diff
@@ -11,15 +11,15 @@ See [Parse section configurations](parse-section)
 This is regexp and time format patterns of this plugin:
 
     :::text
-    expression /^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*) +\S*)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)")?$/
+    expression /^(?<remote>[^ ]*) (?<host>[^ ]*) (?<user>[^ ]*) \[(?<time>[^\]]*)\] "(?<method>\S+)(?: +(?<path>[^\"]*) +\S*)?" (?<code>[^ ]*) (?<size>[^ ]*)(?: "(?<referer>[^\"]*)" "(?<agent>[^\"]*)" (?<http_x_forwarded_for>[^ ]*))?$/
     time_format %d/%b/%Y:%H:%M:%S %z
 
-`remote`, `user`, `method`, `path`, `code`, `size`, `referer` and `agent` are included in the event record. `time` is used for the event time.
+`remote`, `user`, `method`, `path`, `code`, `size`, `referer`, 'agent' and `http_x_forwarded_for` are included in the event record. `time` is used for the event time.
```

Comment:
I noticed that you mixes backticks and quotes in this line.

![2018-04-23-142548_610x58_scrot](https://user-images.githubusercontent.com/8974561/39108255-4949579c-4702-11e8-9cc4-ed96ccd58ff2.png)

Could you update this line to use backticks properly?